testdata/test.txt

```diff
@@ -0,0 +1,22 @@
+Out of the night that covers me,
+      Black as the pit from pole to pole,
+I thank whatever gods may be
+      For my unconquerable soul.
+
+In the fell clutch of circumstance
+      I have not winced nor cried aloud.
+Under the bludgeonings of chance
+      My head is bloody, but unbowed.
+
+Beyond this place of wrath and tears
+      Looms but the Horror of the shade,
+And yet the menace of the years
+      Finds and shall find me unafraid.
+
+It matters not how strait the gate,
+      How charged with punishments the scroll,
+I am the master of my fate,
+      I am the captain of my soul. 
+
+Invictus - By William Ernest Henley
```

Comment:
I still insist that generating random data for your tests, rather than distributing a potentially cherry-picked set along with the repo, is stronger. You can just generate the data in memory, then compare the downloaded data that you push through a mock response to that.